shell/browser/native_window.cc

```diff
@@ -582,6 +582,22 @@ const views::Widget* NativeWindow::GetWidget() const {
   return widget();
 }
 
+base::string16 NativeWindow::GetAccessibleWindowTitle() const {
+  if (accessible_title_.empty()) {
+    return views::WidgetDelegate::GetAccessibleWindowTitle();
+  }
+
+  return base::UTF8ToUTF16(accessible_title_);
```

Comment:
Is there a reason we can't store this as `base::string16`.  Storing as `std::string` just to convert to `string16` every time _something_ in chrome calls this getter seems a bit rough

Reply:
The only reason is that I believe I've seen something similar somewhere else. I've switched to storing it as `base::string16`.